### PR TITLE
[mypy] ignore 2

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-10508)

## Description
ignore mypy errors in

- CrowdStrikeFalconStreamingV2
- FeedCyCognito
- GoogleCloudTranslate
- GoogleKeyManagementService
- GoogleKubernetesEngine
- GoogleKubernetesEngine_test
- TAXII2Server

### CODE REVIEW

File: Packs/GoogleKeyManagementService/Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py [hunk 1]
```diff
@@ -352,7 +351,7 @@ def get_update_command_body(args: Dict[str, Any], update_mask: List) -> Dict:
 
     if 'purpose' in update_mask:
         # Add purpose enum to body
-        body['purpose'] = kms.CryptoKey.CryptoKeyPurpose[args.get('purpose')].value
+        body['purpose'] = kms.CryptoKey.CryptoKeyPurpose[args['purpose']].value  # type: ignore[misc]
```
Comment (kobymeir) [inline Packs/GoogleKeyManagementService/Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py:354]: get -> [] 
not sure we want to do this change

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py [hunk 2]
```diff
@@ -302,10 +301,10 @@ def gcloud_clusters_list_command(client: ClusterManagerClient, project: str, zon
         dict: Cluster raw response.
     """
     # Query and gPRC unpack
-    raw_response_msg: Message = client.list_clusters(project_id=project,
-                                                     zone=zone,
-                                                     timeout=API_TIMEOUT)
-    raw_response_dict: dict = MessageToDict(raw_response_msg._pb)  # type: ignore[attr-defined]
+    raw_response_msg = client.list_clusters(project_id=project,
```
Comment (kobymeir) [inline Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py:304]: why did you remove the typing?

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py [hunk 3]
```diff
@@ -336,10 +335,10 @@ def gcloud_clusters_describe_command(client: ClusterManagerClient, project: str
         dict: Cluster raw response.
     """
     # Query and gPRC unpack
-    raw_response_msg: Message = client.get_cluster(cluster_id=cluster,
-                                                   project_id=project,
-                                                   zone=zone,
-                                                   timeout=API_TIMEOUT)
+    raw_response_msg = client.get_cluster(cluster_id=cluster,
```
Comment (kobymeir) [inline Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py:338]: same q

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: Packs/GoogleKeyManagementService/Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py (hunk 1 above)
Comment (shmuel44) [reply Packs/GoogleKeyManagementService/Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py:354]:
```
if 'purpose' in update_mask:
```

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py (hunk 3 above)
Comment (shmuel44) [reply Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py:338]: wrong type, the type is `Cluster`

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py (hunk 2 above)
Comment (shmuel44) [reply Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py:304]: wrong type, the type is `ListClustersResponse`

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py [hunk 4]
```diff
@@ -638,7 +638,8 @@ def parse_manifest_and_object_args() -> tuple:
             datetime.strptime(added_after, UTC_DATE_FORMAT)
     except ValueError:
         try:
-            datetime.strptime(added_after, STIX_DATE_FORMAT)
+            if added_after:
```
Comment (dorschw) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py:641]: There won't be a ValueError unless the `if added after` in L637 evaluates to `True`. Is L641 mypy's requirement?

Comment (shmuel44) [reply Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py:641]:
```
Argument 1 to "strptime" of "datetime" has incompatible type "str | None";
expected "str"  [arg-type]
                datetime.strptime(added_after, STIX_DATE_FORMAT)
```

**Comment by shmuel44** [comment]
@kobymeir need force merge failed on DO106, RN106

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py [hunk 5]
```diff
@@ -502,27 +501,27 @@ def gcloud_clusters_set_master_authorized_network_command(client: ClusterManager
...
+        cidrs: Comma separated list of CIDRs 192.160.0.0/24,10.0.0.0/24,
 
     Returns:
         str: Human readable.
         dict: Operation entry context.
         dict: Operation raw response.
     """
     # Perform cluster update
-    update = {
+    update: ClusterUpdate = {
```
Comment (kobymeir) [inline Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py:512]: i don't understand? it's a dict why force it's type

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py (hunk 5 above)
Comment (shmuel44) [reply Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py]: done in https://github.com/demisto/content/pull/36013/commits/44fce87cd0fc072761576f2645d4bcc3fd59b0f1

**Review by kobymeir — APPROVED** [review]
